### ISSUE TITLE
OpenGraphData fields are not visible in ObjC

### ISSUE THREAD

**Issue by Luten** [marty-suzuki/URLEmbeddedView#69]
https://github.com/marty-suzuki/URLEmbeddedView/blob/9750da46e0d1eeae55cf7fcbb618ed0bbc8e488b/Example/URLEmbeddedViewSample/OGObjcSampleViewController.m#L54

This example doesn't compile with error:
`Property 'siteName' not found on object of type 'OpenGraphData *'`

Looks like this will fix the issue: https://stackoverflow.com/a/45656756/1321401

**Comment by marty-suzuki** [comment]
Hi, @Luten !

What version do you build?
I've tried building latest master branch, and wroks fine with Xcode 9.4.1

**Comment by Luten** [comment]
I'm building with Xcode 9.2, swift4.0 (upgrade not allowed yet for business reasons)
Framework integrated via cocoaPods:
```
use_frameworks!
target 'MyTarget' do
    pod 'URLEmbeddedView'
end

**Comment by Luten** [comment]
Update:
pod install says:
`Using URLEmbeddedView (0.14.0)`

**Comment by marty-suzuki** [comment]
Example is not used cocoapods.
Please use carthage.

**Comment by marty-suzuki** [comment]
‘@objc’ attribute is not matter because it had already implemented.
Don’t you look implementation?
https://github.com/marty-suzuki/URLEmbeddedView/blob/master/URLEmbeddedView/Core/Entity/OpenGraphData.swift

**Comment by marty-suzuki** [comment]
I've downloaded Xcode 9.2 and tried to build URLEmbeddedView Example with Carthage.
It works fine.

**Comment by Luten** [comment]
Your documentation says it should work with cocoapods too.

**Comment by marty-suzuki** [comment]
What version of README did you see?

Latest README is this.

> To run the example project, clone the repo, and run carthage update from the Example directory first.

It have written clearly at https://github.com/marty-suzuki/URLEmbeddedView#usage

**Comment by Luten** [comment]
> @objc attribute is not matter because it had already implemented.
> Don’t you look implementation?
> https://github.com/marty-suzuki/URLEmbeddedView/blob/master/URLEmbeddedView/Core/Entity/OpenGraphData.swift

This is implemented in newer versions. 
In `0.14.0` wich is last for Swift 4.0 - it is not implemented.

> What version of README did you see
I don't run your example, I told that code from your example doesn't compile because object properties are not visible 

Looks like 0.14 version was not updated for swift 4.0

**Comment by marty-suzuki** [comment]
> I didn't run your example, I said that same code from your example

Sorry, Now I understand your correct situation.

> In 0.14.0 wich is last for Swift 4.0 

Your wrong. I've succeed to run  with Xcode 9.0 at latest version.
Don't you try install pod with `pod install --repo-update`?

**Comment by Luten** [comment]
> Don't you try install pod with pod install --repo-update?
Strangely pods didn't update to latest available version automatically, set manually to 0.15.3 (we need ios8) - now it works.
Great thanks!
